dev-docs/bidders/33across.md

```diff
@@ -4,14 +4,154 @@ title: 33Across
 description: Prebid 33Across Bidder Adapter
 hide: true
 biddercode: 33across
+media_types: banner, video
 gdpr_supported: true
+schain_supported: true
+usp_supported: true
 ---
-
+### Prebid Server Note:
+The 33Across Adapter in Prebid Server does not support multi-impression requests, and will respond with a 400 level response if more than one impression is present in `$.imp[]` of the request.
```

Comment:
Are you saying that PBS fails the whole request if 33across gets an imp with both banner and video? That seems extreme. Shouldn't the 33across adapter just choose one of them and run with that? FWIW, that's what the Rubicon adapter does -- if it sees both banner and video, it chooses video.

Reply:
Not quite, what the note is saying is that our exchange server does not support SRA and will respond with a 400. We do support multi-format within the same `imp` however.